package.json

```diff
@@ -42,8 +42,8 @@
     "format": "prettier --write .",
     "format:check": "prettier --check .",
     "lint": "yarn format:check && yarn lint:ts && yarn lint:js",
-    "lint:ci": "yarn format:check && yarn lint:ts && yarn lint:js --quiet",
-    "lint:js": "eslint --cache .",
+    "lint:ci": "yarn format:check && yarn lint:ts && yarn lint:js",
```

Comment:
making a note to make sure you want to keep these changes (not sure if it was while you were testing things)

Reply:
Yeah so the --quiet flag previously wasn't doing anything. Attaching it on the js lint silences warnings (which I prefer and it speeds up linting by a significant amount since the console doesn't have to log 1000+ warnings).

TLDR: This was intentional